keyboards/monarch/keymaps/iso/keymap.c

```diff
@@ -17,28 +17,62 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 #include QMK_KEYBOARD_H
 
+// Artificial delay added to get media keys to work in the encoder
+#define MEDIA_KEY_DELAY 10
+
 const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
   [0] = LAYOUT_iso(
       KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, KC_BSPC, KC_MUTE,
       KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_ENT,           KC_PGUP,
       KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_NUHS,                   KC_PGDN,
       KC_LSFT, KC_NUBS, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT,          KC_UP,
-      KC_LCTL, KC_LGUI, KC_LALT,                                     KC_SPC,                             MO(1),            KC_LEFT, KC_DOWN, KC_RGHT
+      KC_LCTL, KC_LGUI, KC_LALT,                                     KC_SPC,                             MO(1),            KC_LEFT, KC_DOWN, KC_RGHT,
+      _______, _______
   ),
   [1] = LAYOUT_iso(
       RESET,   KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  _______, _______, _______,
       _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,
       _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,                   _______,
       _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          BL_ON,
-      _______, _______, _______,                                     BL_TOGG,                            _______,          BL_DEC,  BL_OFF,  BL_INC
+      _______, _______, _______,                                     BL_TOGG,                            _______,          BL_DEC,  BL_OFF,  BL_INC,
+      _______, _______
   )
 };
 
-bool encoder_update_user(uint8_t index, bool clockwise) {
+void encoder_update_user(uint8_t index, bool clockwise) {
```

Comment:
Using the code thats in place works perfectly for the encoder and the whole board. I haven't had any issues from any users. Does it really need to be moved?

Reply:
And I'm sorry if that came across negatively, it was more a question. I wasn't meaning to be confrontational or anything at all, I promise. Just trying to understand more. Since I'm not really sure where exactly it should go. Thank you for all your help.

Reply:
No worries.  

The reason I suggested/asked for it to be moved to the keyboard's c file, is so that it works with the configurator, or the like. 

but at the very least, it MUST be `bool`.  the functions in `master` expect them to be, and if they're not, it will fail to compile. 

If you're not seeing that locally, it could be because you have an outdated repo.  Nothing wrong with that, but it can cause issues when testing.

If you need help with updating, the docs go over how.

Reply:
```suggestion
bool encoder_update_user(uint8_t index, bool clockwise) {
```